src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -729,10 +730,10 @@ public void onLoad(FlowExecutionOwner owner) throws IOException {
         try {
             if (isComplete()) {
                 if (done == Boolean.TRUE && !super.isComplete()) {
-                    LOGGER.log(Level.WARNING, "Completed flow without FlowEndNode: "+this+" heads:"+getHeadsAsString());
+                    LOGGER.log(Level.INFO, "Completed flow without FlowEndNode: "+this+" heads:"+getHeadsAsString());
```

Comment:
Certainly _sounds_ like it should be a warning to me, but maybe I am missing something?